server/src/test/java/org/elasticsearch/cluster/coordination/JoinStatusTests.java

```diff
@@ -47,16 +47,16 @@ private JoinStatus mutateJoinStatus(JoinStatus originalJoinStatus) {
                 return new JoinStatus(
                     originalJoinStatus.remoteNode(),
                     originalJoinStatus.term(),
-                    randomAlphaOfLengthBetween(0, 30),
+                    randomValueOtherThan(originalJoinStatus.message(), () -> randomAlphaOfLengthBetween(0, 30)),
                     originalJoinStatus.age()
                 );
             }
             case 3 -> {
                 return new JoinStatus(
                     originalJoinStatus.remoteNode(),
                     originalJoinStatus.term(),
-                    randomAlphaOfLength(30),
-                    randomNonNegativeTimeValue()
+                    originalJoinStatus.message(),
+                    ESTestCase.randomValueOtherThan(originalJoinStatus.age(), this::randomNonNegativeTimeValue)
```

Comment:
Is the `ESTestCase` necessary here?

Reply:
No, not sure why IntelliJ put that there 😕